integration/client_test.go

```diff
@@ -100,38 +100,41 @@ func TestEmptyGetAndUpdate(t *testing.T) {
 	authorizedKeys3 := []*keyspb.PublicKey{authorizedKey2}
 
 	for _, tc := range []struct {
+		desc           string
 		want           bool
 		insert         bool
 		ctx            context.Context
 		userID         string
 		signers        []signatures.Signer
 		authorizedKeys []*keyspb.PublicKey
 	}{
-		{false, false, context.Background(), "noalice", signers1, authorizedKeys1},                // Empty
-		{false, true, GetNewOutgoingContextWithFakeAuth("bob"), "bob", signers1, authorizedKeys1}, // Insert
-		{false, false, context.Background(), "nocarol", signers1, authorizedKeys1},                // Empty
-		{true, false, context.Background(), "bob", signers1, authorizedKeys1},                     // Not Empty
-		{true, true, GetNewOutgoingContextWithFakeAuth("bob"), "bob", signers1, authorizedKeys1},  // Update
-		{true, true, GetNewOutgoingContextWithFakeAuth("bob"), "bob", signers2, authorizedKeys2},  // Update, changing keys
-		{true, true, GetNewOutgoingContextWithFakeAuth("bob"), "bob", signers3, authorizedKeys3},  // Update, using new keys
+		{"Empty", false, false, context.Background(), "noalice", signers1, authorizedKeys1},
```

Comment:
This is pushing the limit of what's readable and maintainable in a test table full of one-liners (and no labels!)

Reply:
Added labels :-)